### ISSUE TITLE
Walking not reflected in App

### ISSUE THREAD

**Issue by rafagjimenez** [huacnlee/PokemonGoMove#59]
Both servers are synced.
I can even "walk" the red dot in the upper screen and is reflected in the bottom screen but in the application Im stucked in the default location wher I start the example.py  script  but If I "walk in the application either moving arrows/dragging the pointer/etc" its not reflected in the Application. Im stucked in the initial spawning point.

**Comment by SaipanDamashii** [comment]
Same here, hope someone can help us soon...

**Comment by yqrashawn** [comment]
You need to make sure the foo app is running on your iPhone and the location stimulation is on in Xcode debug menu. @rafagjimenez @SaipanDamashii

**Comment by diddy81** [comment]
try this from issue [40](https://github.com/huacnlee/PokemonGoMove/issues/40)

**Comment by SaipanDamashii** [comment]
@yqrashawn Yes, I know this.  It still doesn't work, my character still isn't moving when I move the map around...

**Comment by yqrashawn** [comment]
@SaipanDamashii Is there any other location faker on your iPhone or have you installed any tweak which may fake your location in Pokemon app. Turn them off.

**Comment by SaipanDamashii** [comment]
Nope. Nothing. This is the first time I've even heard that we can do this.

**Comment by diddy81** [comment]
can you move around with the arrow keys??
if the answer is yes then try to use you local ip instead of localhost for the localhost:3001
